vnext/Microsoft.ReactNative.Managed.CodeGen/Microsoft.ReactNative.Managed.CodeGen.csproj

```diff
@@ -30,6 +30,16 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.7.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.7.0" />
     <PackageReference Include="RuntimeContracts" Version="0.3.0" />
+
+    <!-- 
+      NuGet decides to pull in packages with issues. There is no seeming way to diagnose why NuGet is pulling in those versions. 
+      Running restore with -Verbosity Detailed and environment variables:
+       * NUGET_RESTORE_MSBUILD_ARGS=/bl:c:\temp\nuget.binlog
+       * NUGET_RESTORE_MSBUILD_VERBOSITY=diag
+      resulted in no mention of version 4.3.0 anywhere until NuGet just starts downloading it.
+      Therefore manually overriding the version so NuGet uses the latest.
+    -->
+     <PackageReference Include="System.Private.Uri" Version="4.3.2" />
```

Comment:
Any idea how component governance recognizes NuGet dependencies? I know for yarn deps, its all static. Wondering if it's going to chase transitive dependencies and find the old one still.

Reply:
It looks like it looks here:

```
/s/vnext/packages/system.private.uri/4.3.0/system.private.uri.4.3.0.nupkg
/s/vnext/packages/system.private.uri/4.3.0/system.private.uri.nuspec
```

The recommended fix is:

```
Upgrade to version Microsoft.NETCore.App.nupkg - 2.1.8,2.2.2;System.Private.Uri.nupkg - 4.3.1
```

Reply:
But the problem here is that the both are NetCore3.1 . I suspect a Nuget Quirk. I tried to find some diagnostics in the nuget restore command. Per the comment in the code, but NuGet is quite opaque here and doesn't provice enough information to see why it is included or why it resolved to 4.3.0. The only recourse one has fo rnuget is to add explicit dependenices, which I did.